deps/mbedtls.mk

```diff
@@ -7,12 +7,8 @@ else
 endif
 MBEDTLS_URL = https://tls.mbed.org/download/$(MBEDTLS_SRC).tgz
 
-MBEDTLS_OBJ_SOURCE := $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedtls.$(SHLIB_EXT) \
-			$(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedx509.$(SHLIB_EXT) \
-			$(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedcrypto.$(SHLIB_EXT)
-MBEDTLS_OBJ_TARGET := $(build_shlibdir)/libmbedtls.$(SHLIB_EXT) \
-			$(build_shlibdir)/libmbedx509.$(SHLIB_EXT) \
-			$(build_shlibdir)/libmbedcrypto.$(SHLIB_EXT)
+MBEDTLS_OBJ_SOURCE := $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/library/libmbedcrypto.$(SHLIB_EXT)
+MBEDTLS_OBJ_TARGET := $(build_shlibdir)/libmbedcrypto.$(SHLIB_EXT)
```

Comment:
ping. not critical, but for the record would be good to have answers written down to these last minor clarifications while it's fresh in memory